src/MongoDB.Analyzer/Core/Poco/Data/City.json

```diff
@@ -0,0 +1,24 @@
+﻿{
+    "city": [
+        "New York City",
```

Comment:
Done. Only issue with plural names is that, however, if we have a JSON field called "Names" and we have a POCO with a property "Name", since we are doing POCOPropertyName.Contains(JsonFieldName), it won't catch "Name"